src/test/scala/nl.knaw.dans.easy.solr4files/AppSpec.scala

```diff
@@ -73,7 +70,9 @@ class AppSpec extends TestSupportFixture {
   "update" should "call the stubbed solrClient.request" in {
     initVault()
     assume(canConnectToEasySchemas)
-    val result = new MockedAndStubbedApp().update(store, uuidCentaur)
+    val result = new TestApp() {
+      override val solrClient: SolrClient = stubbedSolr
```

Comment:
If you make a function to create this 'stubbedApp' you can call that instead of this override val stuff...